canonical-web-and-design/ubuntu.com
# Remove useless fixtures

## Done

- Remove testing fixtures
- Add loading releases in migration

## QA

- Check out this feature branch
- `rm usn.sqlite3`
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://172.17.0.2:8001/security/notices
- Drop down should show the releases.
